src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2159,7 +2159,12 @@ def aks_get_credentials(cmd, client, resource_group_name, name, admin=False,
     # in which case we ignore the KUBECONFIG variable
     # KUBECONFIG can be colon separated. If we find that condition, use the first entry
     if "KUBECONFIG" in os.environ and path == os.path.join(os.path.expanduser('~'), '.kube', 'config'):
-        path = os.environ["KUBECONFIG"].split(":")[0]
+        kubeconfig_path = os.environ["KUBECONFIG"].split(":")[0]
+        if kubeconfig_path:
+            logger.info("The default path '%s' is replaced by '%s' defined in KUBECONFIG.", path, kubeconfig_path)
+            path = kubeconfig_path
+        else:
+            logger.warning("Invalid path '%s' defined in KUBECONFIG.", kubeconfig_path)
```

Comment:
Why just print the warning log instead of raising an exception?

Reply:
Personally, I think there are some usage scenarios where the user has multiple kubeconfig files, then sets the environment variable `KUBECONFIG` to point to a specific kubeconfig file to perform some operatations, and sometimes leaves `KUBECONFIG` empty to operate with the kubeconfig located in the default path (`~/.kube/config`).

I hope this change is that when the user sets `KUBECONFIG` to be empty, the kubeconfig of the cluster can also be successfully downloaded to the default location.

More discussions are welcome：）

Reply:
Make sense!